scss/_patterns_cta.scss

```diff
@@ -0,0 +1,19 @@
+@import 'settings';
+
+@mixin vf-p-cta {
+  .p-cta-block {
+    // same as %section-padding--shallow
+    padding-bottom: $spv--x-large;
+
+    @media screen and (min-width: $breakpoint-large) {
+      // CTA is shifted to middle of the page on larger displays
+      margin-left: auto;
+      width: 50%;
```

Comment:
It's not middle of the page. It needs to be aligned with the 7th column of the grid.

I don't think we can easily calculate that as a percentage, it would need to acutally implement the grid layout.

Reply:
And I don't think it should do that by default. Because this very much restricts the placement of this component. We assume it will always take the whole row of content.

This needs to be independent enough to be placed anywhere, in any container. Including inside other columns.

That's why we need to first define what are the boundries of this component and how it is expected to be used.